examples/nat_topo.py

```diff
@@ -230,11 +237,15 @@ def sendInt(net, name, **args):
     # Set up the parser
     parser = argparse.ArgumentParser(description="Test the Pax NAT implementation.")
     parser.add_argument("action", choices=["run", "test"], nargs="?", default="run")
-    parser.add_argument("--no-X", help="don't launch additional windows'", action="store_false", dest="X_windows")
+    parser.add_argument("--no-X", help="don't launch additional windows", action="store_false", dest="X_windows")
+    parser.add_argument("--hold-open", help="leave xterm windows open", action="store_true", dest="hold_open")
+    parser.add_argument("--cli-first", help="provide cli access before running the tests", action="store_true", dest="cli_first")
```

Comment:
The user gets access to the Mininet CLI after the network is setup, but before Pax is started. They can finish the CLI with ^D and then the tests will start. I will clarify the help text.